app/models/types_de_champ/commune_type_de_champ.rb

```diff
@@ -27,6 +27,31 @@ def champ_value(champ)
     champ.code_postal? ? "#{champ.name} (#{champ.code_postal})" : champ.name
   end
 
+  def columns(procedure:, displayable: true, prefix: nil)
+    super.concat(
+      [
+        Columns::JSONPathColumn.new(
+          procedure_id: procedure.id,
+          stable_id:,
+          tdc_type: type_champ,
+          label: "#{libelle_with_prefix(prefix)} (Code Postal)",
+          jsonpath: '$.code_postal',
+          displayable:,
+          type: :text
+        ),
+        Columns::JSONPathColumn.new(
+          procedure_id: procedure.id,
+          stable_id:,
+          tdc_type: type_champ,
+          label: "#{libelle_with_prefix(prefix)} (Département)",
```

Comment:
Pour être iso avec `AddressableConcern`


```suggestion
          label: "#{libelle_with_prefix(prefix)} - code postal (5 chiffres)",
          jsonpath: '$.code_postal',
          displayable:,
          type: :text
        ),
        Columns::JSONPathColumn.new(
          procedure_id: procedure.id,
          stable_id:,
          tdc_type: type_champ,
          label: "#{libelle_with_prefix(prefix)} - département",
```

Reply:
embarqué !